WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -38,13 +44,22 @@ public class SimulationApiModel
         [JsonProperty(PropertyName = "EndTime", NullValueHandling = NullValueHandling.Ignore)]
         public string EndTime { get; set; }
 
+        [JsonProperty(PropertyName = "StopTime", NullValueHandling = NullValueHandling.Ignore)]
+        public string StopTime { get; set; }
+
         [JsonProperty(PropertyName = "DeviceModels")]
         public IList<SimulationDeviceModelRef> DeviceModels { get; set; }
 
+        [JsonProperty(PropertyName = "TotalMsgs", NullValueHandling = NullValueHandling.Ignore)]
+        public int TotalMessagesSent { get; set; }
+
+        [JsonProperty(PropertyName = "AvgMsgs", NullValueHandling = NullValueHandling.Ignore)]
+        public double AverageMessagesSent { get; set; }
```

Comment:
I think the scope of the PR is growing from multiple simulations to something else?  also, the average should have a reference, e.g. is this per second/per minute?  I'd like to discuss the new API before merging this PR, to see how it looks like and what's the expected behavior in edge cases. For instance, it's not clear which fields are readonly (statistics) and which not. A client won't know which fields are required when creating a simulation, and which ones are not. As a side note, I was thinking to move all the statistics into an object, something like:
```json
{
  ...
  "Statistics": {
    "TotalDevices": 20000,
    "DevicesCurrentlyConnected": 15200,
    "AverageMessagesPerSecond": 2200,
    ...
  }
...
```

Reply:
Again, this is interim solution to store these values and will change with Hugh's work. For now I will move statistics under one object. 

Let' s chat tomorrow